en/ha.rst

```diff
@@ -3515,7 +3515,7 @@ checksumdb
     
 .. option:: -n, --table-name=STRING
 
-    You can specify a table name to save checksum results. (default: db_ha_checksum)
```

Comment:
Shouldn't it be mentioned that the owner name should be specified? 
I'm not sure as I don't know how the other utilities will change. But I mentioned it when describing flashback.

Reply:
I will apply that like below;

`You can specify a table name with owner name(owner_name.table_name) to save checksum results.`

Reply:
My opinion is that the previous sentence is better for me. If you want to explain the input format, then it is better to add an additional sentence for it.

Reply:
@hornetmj As far as I was told in the previous meeting, this kind of change will be done by who is in charge of the user schema, won't it? (Actually, I thought the description for the default value will be changed with the change then by him)

Reply:
@wrlawodms In this regard, I have added @youngjinj as a reviewer.

Reply:
The owner name can only be "dba".
If we need to specify this in usage, we need to ask the user to input "dba.<table_name>".

I think sentences should be short.
So I think it's better to add a description in another sentence.
```
The table name must include the owner name.
The owner name can only be dba.
```

Reply:
AS-IS
`You can specify a table name to save checksum results. (default: dba.db_ha_checksum)`

TO-BE
`You can specify a table name with the owner name to save checksum results.(the owner name can only be dba) The table name must include the owner name . (default: dba.db_ha_checksum)`

@hornetmj @youngjinj @wrlawodms @joohok 
How about this change?

Reply:
1. What happens if a non-dba user is specified?
2. How about just getting only a table name and attaching the dba internally (which is done automatically as far as I know)?

Reply:
As far as I remember, tables can be created by users other than the DBA.
how about
`You can specify a table name (owner_name.class_name) to save checksum results. (default: dba.db_ha_checksum)` ?

Reply:
@wrlawodms 

1. It does not work with -n option with error message.
2. Yes, It is already applied

@joohok 

- checksumdb result table "db_ha_checksum" or "user_defined" is set owner to dba only.

Reply:
If it works like (2), why does it require the user to input a dba or user name, which seems meaningless?

Reply:
Do you tell about -n option? In my opinion, it seems to relate with only table name.

Reply:
I prefer TO-BE1 with some corrections if you ask:
```
You can specify a table name to save checksum results. The table name must include the owner's name, which must be dba. (default: dba.db_ha_checksum)
```
but I'm OK with whichever version includes enough information.

Reply:
I agree with @wrlawodms.

Users can only specify the owner name as DBA.
Nevertheless, it is a problem for the user to enter the owner name.

This was to unify the format for table input.
And when inconveniences or problems like this are found, we decided to improve them.

So I think it should be included as an improvement in the next patch or release.

Reply:
A legacy bug is founded. The checksumdb utility never complains if the already existing table is used with -n option. But, there is no result from checksumdb and an error message is found in the error log.

### Repro.
* 10.2 is used because I want to show that it is a legacy bug.
* In this case, the checksumdb utility has to return a correct error message.
```hornetmj1: [~]
$ csql -u dba -c "create table a123 (c1 int)" demodb
Execute OK. (0.001956 sec) Committed.
hornetmj1: [~]
$ csql -u dba -c "desc a123" demodb

=== <Result of SELECT Command in Line 1> ===

  Field                 Type                  Null                  Key                   Default               Extra
====================================================================================================================================
  'c1'                  'INTEGER'             'YES'                 ''                    NULL                  ''

1 row selected. (0.004074 sec) Committed.
hornetmj1: [~]
$ cubrid checksumdb -n a123 demodb
hornetmj1: [~]
$ csql -u dba -c "select * from a123" demodb

=== <Result of SELECT Command in Line 1> ===

There are no results.
0 row selected. (0.004462 sec) Committed.
hornetmj1: [~]
$ cd CUBRID/log/
hornetmj1: [~/CUBRID/log]
$ grep -r 'a123' *
cubrid_utility.log:22-05-24 11:08:37.798 (10546) cubrid checksumdb -n a123 demodb
demodb_checksumdb.err:checksumdb: Invalid checksum table [a123] exists (error code: -202).
server/demodb_20220524_1049.err:Started to update statistics (class "a123", oid : 0|204|11).
server/demodb_20220524_1049.err:Finished to update statistics (class "a123", oid : 0|204|11, error code : 0).```

Reply:
@eido5 I chose the TO-BE1 option. But I think that the meaning of the two sentences is equal. So, It is a good choice to ask @charischau correct the sentences naturally.

Reply:
the TO-BE1 option: 
You can specify a table name to save checksum results. The table name must include the owner name. The owner name can only be dba. (default: dba.db_ha_checksum)

In my opinion, we can shorten the sentence into: 
**You can specify a table name to save checksum results. The table name must include the owner's name which can only be dba. (default: dba.db_ha_checksum).** 

*if you want to emphasize 'Users can only specify the owner name as DBA.', you can capitalize the 'ONLY' as: 
You can specify a table name to save checksum results. The table name must include the owner's name which can ONLY be dba. (default: dba.db_ha_checksum). 

@wrlawodms , in your suggestion,
You can specify a table name to save checksum results. The table name must include the owner's name, which must be dba. (default: dba.db_ha_checksum). 

the comma between 'owner's name' and 'which' might cause confusion about what is the second half-sentence referred to.

Reply:
@charischau Thank you for your suggestion!

`You can specify a table name to save checksum results. The table name must include the owner's name which can only be dba. (default: dba.db_ha_checksum).`

@hornetmj @youngjinj @wrlawodms @joohok 
I will fix the manual as above sentences. 
Would you check the sentences until today's worktime?
If you are okay, would you push the 'Thumbs up' emotion?